packages/libs/eda/src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -318,6 +318,13 @@ export function HistogramFilter(props: Props) {
     uiState.dependentAxisLogScale
   ) as NumberRange;
 
+  const fgSummaryStatsMin = formatStatValue(fgSummaryStats?.min, variable.type);
```

Comment:
Let's move these right below the definition of `fgSummaryStats` if possible? That way all of the foreground summary stats things get done right next to each other.

Reply:
Hi @asizemore thank you for your suggestions. 👍 yes it makes sense, so I will move them